.github/workflows/ci.yml

```diff
@@ -45,15 +45,15 @@ jobs:
 
   run_checks:
     name: Run checks and tests
-    runs-on: 'ubuntu-latest'
     strategy:
       matrix:
         rust_toolchain_version: ["1.79", "1.80", "1.81", "1.82"]
         # FIXME: currently not available for 5.0.0.
         # It might be related to boxroot dependency, and we would need to bump
         # up the ocaml-rs dependency
         ocaml_version: ["4.14"]
-        node: [20]
```

Comment:
why does it need to remove the node entry?

Reply:
Oh no, that's an error... Nice!

Reply:
Fixed in https://github.com/o1-labs/proof-systems/pull/3133/commits/69a5df794e45da69910a42821401ae3a333bacf8.